app/components/Quest/Loading.tsx

```diff
@@ -0,0 +1,6 @@
+export default function Loading({ color="sky" }) {
```

Comment:
We should check with Allan to use a color from figma.

Reply:
added the classic green for now